### ISSUE TITLE
Navigation using arrows does not work after switching tabs

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#107747]
Testing #107610

- Switch from user to workspace settings tab
- Use arrows to navigate
🐛  Does not work

**Comment by roblourens** [comment]
I don't understand. Where is focus when this is happening? You would have to go back to the settings list

**Comment by sandy081** [comment]
Please check the attached gif - at the end after navigating to workspace settings tab and pressing enter, I cannot navigate settings using arrow keys.

![Kapture 2020-09-30 at 09 08 07](https://user-images.githubusercontent.com/10746682/94653905-959c6180-02fc-11eb-96a1-036da9a5a27c.gif)

**Comment by roblourens** [comment]
Are you expecting "enter" to move focus from the tabs to the settings editor or "down" to move focus back?

**Comment by sandy081** [comment]
I expect after `enter`, down arrow to navigate settings.
